api/src/main/resources/workbench.yaml

```diff
@@ -795,6 +795,76 @@ paths:
           schema:
             $ref: "#/definitions/CloneWorkspaceResponse"
 
+  # Notebooks ############################################################################
+
+  /v1/notebooks/{workspaceNamespace}/{workspaceId}/renameNotebook:
```

Comment:
Notebooks are hierarchically under workspaces, so this should probably be something more like:

`/v1/workspaces/{workspaceNamespace}/{workspaceId}/notebooks/{notebookName}/rename`

e.g. see the cohort URLs